libsolidity/ast/Types.cpp

```diff
@@ -3004,14 +3004,16 @@ MemberList::MemberMap MagicType::nativeMembers(ContractDefinition const*) const
 	switch (m_kind)
 	{
 	case Kind::Block:
-		return MemberList::MemberMap({
-			{"coinbase", make_shared<IntegerType>(160, IntegerType::Modifier::Address)},
-			{"timestamp", make_shared<IntegerType>(256)},
-			{"blockhash", make_shared<FunctionType>(strings{"uint"}, strings{"bytes32"}, FunctionType::Kind::BlockHash, false, StateMutability::View)},
-			{"difficulty", make_shared<IntegerType>(256)},
-			{"number", make_shared<IntegerType>(256)},
-			{"gaslimit", make_shared<IntegerType>(256)}
-		});
+		{
```

Comment:
In case-statements the `{` usually have the same indentation level as the `case` keyword itself.

Reply:
Did you actually change anything here?

Reply:
No. In the end I did not change anything. These braces are leftovers from a previous commit and I'll remove them.